### DISCUSSION TITLE
Even Validation Method

### DISCUSSION THREAD

**Discussion by adanebehr** [winedarksea/AutoTS#125] [Q&A]
Should setting validation_method='even' force the number of cross validations performed to equal the set num_validations? Assuming that there is more than enough data for the first validation on the most recent data. This was my expectation since the documentation says the 'even' method splits up the data evenly, instead of sequentially stepping backwards in steps forecast_length long through the available data, like the 'backwards' method. Sorry if this is a dumb question. Data science noob here.

**Comment by winedarksea** [comment]
Nope!
`num_validations` + 1 is always the number of trials run (so this was confusing semantics, I defined the initial evaluation as a "test" evaluation while others are "validations" hence total trials is `num_validations` + 1. `validation_method` never changes `num_validations`.

`validation_method` is unique to time series forecasting. With other data science type problems, you just randomly sample the data in N chunks. With time series that doesn't work because you can only sample training data from before the evaluation window, and you can't just randomly sample bits, because having a bunch of holes in your history leads to inaccurate forecasts. `backwards` is almost always the safest bet, progressively removing the most recent data and running a trial up to a certain time, and is the way you will see in most textbooks and blog posts. `seasonal` is backwards but allowing for an extra skip backwards to always be landing on the same time of year. `similarity` attempts to find validation periods that most closely match the current time, this only being particularly useful on large datasets with lots of history. Similarity is something I came up with, you won't see that anywhere else.

Short answer: stick with `backwards` most of the time, and especially recommended as beginner to not worry about anything else.
